src/main/java/cn/nukkit/block/BlockWaterLily.java

```diff
@@ -19,7 +19,8 @@ public BlockWaterLily() {
     }
 
     public BlockWaterLily(int meta) {
```

Comment:
‘meta’ value is redundant here if it has not been used.

Reply:
SupremeMortal did the same
https://github.com/NukkitX/Nukkit/blob/dfecdde7f9c033f1f57daed431277724f01ae78a/src/main/java/cn/nukkit/block/BlockDeadBush.java#L21-L24

Reply:
IIRC, Reflection is used to get the instance and the incorrect constructor will throw an exception.